package.json

```diff
@@ -4,14 +4,14 @@
   "private": true,
   "scripts": {
     "ng": "ng",
-    "clean": "del ./dist",
+    "clean": "del dist",
     "start": "npm run angular:dev:start",
-    "build": "npm-run-all build:ci website:build build:clean",
-    "build:ci": "npm-run-all --parallel format lint clean && npm-run-all build:libs build:apps test",
-    "build:libs": "npm-run-all ui:build icons:build core:build angular:build schematics:build angular:golden",
+    "build": "npm-run-all build:ci website:build",
+    "build:ci": "npm-run-all clean && npm-run-all --parallel format lint && npm-run-all build:libs build:apps build:clean test",
```

Comment:
Is the reason for the multiple `npm-run-all`'s that these commands need to run sequentially? 
There is a way to specify that with npm-run-all
`npm-run-all -s` || `run-s` (shorthand)

Reply:
When we get to `build:libs build:apps build:clean` they have to run in sequentially since the apps depend on the final build output of the libraries. The tests could be ran in parallel with the previous command but did not have any performance benefit since `build:libs build:apps` internally are each building their libs and apps in parallel as well.
I'll clean up the script with the shortcuts

Reply:
Edit: going to clean up the script in my next PR so this can get merged to fix the other issue